README.md

```diff
@@ -106,18 +133,4 @@ Default value: `false`
 - **timeout:**  
 The amount of seconds after a task "end event" can be considered lost.  
 The task "map" is then evicted.  
-The default value is 0, which means no timeout so no auto eviction.
-
-## Need Help?
```

Comment:
After thinking a little about that, I realize that "Need Help" is more a "plugin user" thing than a "plugin developer" thing.
=> I get it back to README.md
Tell me what you think about that.